demo.html

```diff
@@ -69,6 +120,18 @@ <h1 slot="site-title">Calendar Component Demo</h1>
       display="4"
       limit="4">
     </byu-calendar>
+    <hr>
+
+    <h1 class="subheader">Full Page Image Rows</h1>
+    <byu-calendar 
+      title="Upcoming Events"
+      categories="3+4+5"
+      days="7"
+      price="4.5"
+      display="5"
+      limit="4">
+    </byu-calendar>
```

Comment:
We should also add an example WITHOUT a price limitation. To my knowledge, no one has used this filter and it will likely be used in rare cases only.